compiler/mx.compiler/suite.py

```diff
@@ -74,6 +74,13 @@
       "sha1" : "59b64c974662b5cf9dbd3cf9045d293853dd7a51",
     },
 
+    "DACAPO_D3S" : {
+      "urls" : [
+        "http://d3s.mff.cuni.cz/software/benchmarking/files/dacapo-9.12-d3s.jar",
```

Comment:
cc @gilles-duboscq Should this be moved to lafo, as e.g. `JAVA_ALLOCATION_INSTRUMENTER`?

Reply:
We can mirror it on lafo but the d3s url can stay

Reply:
You can now add this URL: https://lafo.ssw.uni-linz.ac.at/pub/graal-external-deps/dacapo-9.12-d3s.jar
It would be more convenient for us if you put the lafo URL first in the list and then the d3s.mff.cuni.cz one.

Reply:
Thanks, I have added lafo as the primary server.